lib/vagrant-berkshelf/action/share.rb

```diff
@@ -22,7 +22,7 @@ def call(env)
             value = chef.config.send(:prepare_folders_config, env[:berkshelf].shelf)
 
             @logger.debug "Setting cookbooks_path = #{value.inspect}"
-            chef.config.cookbooks_path = value
+            chef.config.cookbooks_path += value
```

Comment:
I have updated the code accordingly. `value` is an Array already, so the square brackets around it is unnecessary.